### PR TITLE
Add page-experience.md

### PR DESCRIPTION
**Pull request by nainar**
Add template for Pixi to file issues on Perf WG

This is just the template - we will need to add the placeholders fo where the values go next

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/page-experience.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Cache passes all criteria, Origin fails one or more criteria, Page experience checker had no feedback to provide.
+- Cache fails one or more criteria, Origin passes all criteria
+- Cache and Origin fail one or more criteria, Page experience checker had no feedback to provide.
+
+-->
+
+
+| Metric                  | Field data - Origin | Field data - Cache | Lab data - Origin | Lab data - Cache |
+|-------------------------|---------------------|--------------------|-------------------|------------------|
+| LCP                     |                     |                    |                   |                  |
```
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/page-experience.md:29]: Thoughts on adding `<pass/fail>` options in each row? Could be more clear how to fill this in but risks being too verbose.

Comment (nainar) [reply .github/ISSUE_TEMPLATE/page-experience.md]: Added

File: .github/ISSUE_TEMPLATE/page-experience.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Safe browsing           | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| Mobile-friendliness     | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| Intrusive Interstitials | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+
+Notes
+---
+
+Suspected components in use: 
+<!--
+List components on the page that may be cuasing these issues
```
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/page-experience.md:41]:
```suggestion
List components being used on the page
```

File: .github/ISSUE_TEMPLATE/page-experience.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| CLS                     | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| HTTPS                   | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| Safe browsing           | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| Mobile-friendliness     | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+| Intrusive Interstitials | <pass/fail>         | <pass/fail>        | <pass/fail>       | <pass/fail>      |
+
+Notes
+---
+
+Suspected components in use:
```
Comment (caroqliu) [inline .github/ISSUE_TEMPLATE/page-experience.md:39]:
```suggestion
Components in use: 
```

**Review by caroqliu — COMMENTED** [review]
Made two suggestions about component language -- why not collect all the ones on the page so we have a better big picture understanding of everything at play?

**Pull request merged**